qa/L0_backend_python/python_test.py

```diff
@@ -245,6 +245,22 @@ def test_ensemble(self):
             self.assertTrue(np.allclose(output0, 2 * input_data_0))
             self.assertTrue(np.allclose(output1, 2 * input_data_1))
 
+    def test_unicode(self):
+        model_name = "string"
+        shape = [1]
+        with httpclient.InferenceServerClient("localhost:8000") as client:
+            utf8 = '😀'
+            input_data = np.array([bytes(utf8, encoding='utf-8')], dtype=np.bytes_)
+            inputs = [
+                httpclient.InferInput("INPUT0", shape,
+                                      np_to_triton_dtype(input_data.dtype))
+            ]
+            inputs[0].set_data_from_numpy(input_data)
+            result = client.infer(model_name, inputs)
+            output0 = result.as_numpy('OUTPUT0')
+            self.assertTrue(output0 is not None)
+            self.assertTrue(str(output0[0], encoding='ascii') == 'success')
```

Comment:
Instead of returning success I would pass unicode and have an identity model that returns the same so you can verify that the handling of unicode for outputs is also validated.

Reply:
Fixed. Although, since it is just a blob of bytes I don't think it makes a difference.